src/app/containers/MediaPage/__snapshots__/index.test.jsx.snap

```diff
@@ -163,10 +163,8 @@ Array [
   font-size: 1.75rem;
   line-height: 2rem;
   color: #3F3F42;
-  font-family: ReithSerif,Helvetica,Arial,sans-serif;
```

Comment:
This probably requires passing service etc

Reply:
I think this is valid because the service passed in the test is "amharic" which doesn't have fonts associated with it